core/app/models/spree/adjustment.rb

```diff
@@ -72,9 +52,55 @@ class Adjustment < Spree::Base
     extend DisplayMoney
     money_methods :amount
 
+    def finalize!
+      update_attributes!(finalized: true)
+    end
+
+    def unfinalize!
+      update_attributes!(finalized: false)
+    end
+
+
+    # Deprecated methods
+    def state
+      ActiveSupport::Deprecation.warn "Adjustment#state is deprecated. Instead use Adjustment#finalized?", caller
+      finalized?? "closed" : "open"
```

Comment:
not important, but did you mean to omit the space between the question marks?  it jumped out at me like "finalized???" :)